src/reading.rs

```diff
@@ -241,7 +241,7 @@ impl PageParser {
 			page_siz += *val as u16;
 			// Increment by 1 if val < 255, otherwise by 0
 			self.packet_count += (*val < 255) as u16;
-			self.bi.ends_with_continued = !(*val < 255);
```

Comment:
This piece of code used to be analog to the code above. I think it's better in the old state.

Reply:
revert.